bat/tests/clr-installer-config/run.bats

```diff
@@ -7,28 +7,28 @@ setup() {
   global_setup
 }
 
-@test "Build image using default ister config" {
+@test "Build image using default clr-installer config" {
   mixer $MIXARGS init --clear-version $CLRVER --mix-version 10
   mixer $MIXARGS bundle add editors
 
   # `build image` should fail because the prerequisite commands are not executed.
   # This is intentional in order to reduce test execution time.
-  # The goal is to unit test the creation of ister config file with relevant bundles.
+  # The goal is to unit test the creation of clr-installer config file with relevant bundles.
   run sudo mixer $MIXARGS build image
-  [[ "$status" -eq 1 ]]
-  [[ "$output" =~ "release-image-config.json not found" ]]
+  [[ "$status" -ne 0 ]]
+  [[ "$output" =~ "release-image-config.yaml not found" ]]
   [[ "$output" =~ "Copying image template" ]]
```

Comment:
`make check` is failing because this `[[ "$output" =~ "Copying image template" ]]` is not required anymore.